app/helpers/movie_helper.rb

```diff
@@ -0,0 +1,14 @@
+module MovieHelper
```

Comment:
I don't think this makes sense as a view helper. In fact, I would not even put it in the application (`app/`), rather in `lib/`, as everything this does is something related to knowledge of an external system, and is totally independent to the usage it is currently serving inside the application.

If not for anything else, because we may want this helpers elsewhere (so there are 2 extra levels of coupling unnecessary: this is not just for `movies` (logic for directors is almost the same) nor is necessarily just for views (can be used for backend).

Reply:
Moved to a `ExternalUrls` module under `/lib`.